app/src/main/java/com/example/myapplication/ProfileActivity.kt

```diff
@@ -0,0 +1,150 @@
+package com.example.myapplication
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.*
+
+class ProfileActivity : AppCompatActivity() {
+
+    lateinit var rbMale: RadioButton
+    lateinit var rbFemale: RadioButton
+    private lateinit var edtSecName: EditText
+    private lateinit var edtName: EditText
+    private lateinit var edtOtchestvo: EditText
+    private lateinit var edtBirthdate: EditText
+    private var flagBtnEditProfile: Boolean = false
+    private lateinit var edtGrazhd: EditText
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_profile)
+//        val toolbar = findViewById<Toolbar>(R.id.toolbar)
+        //setSupportActionBar(toolbar);
+/*        supportActionBar?.homeAsUpIndicator = R.drawable.ic_baseline_clear_24
+        supportActionBar?.displayHomeAsUpEnabled = true*/
+        //toolbar!!.title = "Личные данные"
+        //toolbar.setDisplayHomeAsUpEnabled(true)
+
+        val intent = Intent(this, MainActivity::class.java)
+
+/*        toolbar?.apply {
+            setNavigationIcon(R.drawable.ic_baseline_clear_24)
+            setNavigationOnClickListener{
+                startActivity(intent)
+            }
+        }*/
+
+        edtSecName = findViewById(R.id.edtSecondName)
+        edtName = findViewById(R.id.edtName)
+        edtOtchestvo = findViewById(R.id.edtOtchestvo)
+        edtBirthdate = findViewById(R.id.edtBirthdate)
+        edtGrazhd = findViewById(R.id.edtGrazhdanstvo)
+
+        rbMale = findViewById(R.id.rbMale)
+        rbFemale = findViewById(R.id.rbFemale)
+
+        blockAll()
+
+        val btnProfileEdit = findViewById<Button>(R.id.btnProfileEdit)
+        btnProfileEdit.setOnClickListener{
+            if (!flagBtnEditProfile) {
+                unlockAll()
+                flagBtnEditProfile = true
+            }
+            else {
+                blockAll()
+                flagBtnEditProfile = false
+            }
+        }
+
+        val btnProfileBack = findViewById<Button>(R.id.btnBack3)
+        btnProfileBack.setOnClickListener{
+            val intent = Intent(this, MainActivity::class.java)
+            startActivity(intent)
+        }
+
+        val btnGoToView = findViewById<Button>(R.id.btnGoToViewProfile)
+        btnGoToView.setOnClickListener{
+            val intent = Intent(this, ViewProfileActivity::class.java)
```

Comment:
val intent = Intent(this, ViewProfileActivity::class.java).apply {
            putExtra("secName", edtSecName.text.toString())
            putExtra("name", edtName.text.toString())
            putExtra("otchestvo", edtOtchestvo.text.toString())
            putExtra("birthdate", edtBirthdate.text.toString())
            putExtra("grazhd", edtGrazhd.text.toString())
}